aqua/pgAdmin3/Portfile

```diff
@@ -30,34 +31,44 @@ checksums           rmd160  48e732089cd1879f99d1894229a9cc23ed3c0c44 \
                     sha256  9b68b0e3d3f0a261344fb8196825967d1e7acaca1d8cc82c42e12368ca5844ac \
                     size    14788481
 
-wxWidgets.use       wxWidgets-3.0
+if {${os.platform} eq "darwin" && ${os.major} > 11} {
+    wxWidgets.use   wxWidgets-3.0
+} else {
+    wxWidgets.use   wxGTK-3.0
+}
 
 # For unknown reasons pgAdmin3.app/Contents/MacOS/pgAdmin3 still links to
-# libcrypto.1.1.dylib, so the openssl dependency is needed even if pgAdmin3
+# libcrypto.dylib, so the openssl dependency is needed even if pgAdmin3
 # is built with libgcrypt.
 depends_lib         path:lib/pkgconfig/icu-uc.pc:icu \
-                    path:lib/libssl.dylib:openssl \
                     port:libgcrypt \
                     port:libxml2 \
                     port:libxslt \
+                    path:lib/libssl.dylib:openssl \
                     port:${wxWidgets.port}
 
-configure.args      --with-wx=${wxWidgets.prefix} \
-                    --with-wx-version=${wxWidgets.version} \
+configure.args      --enable-appbundle \
+                    --with-arch-${configure.build_arch} \
+                    --with-libgcrypt=${prefix} \
                     --with-libxml2=${prefix} \
                     --with-libxslt=${prefix} \
-                    --with-arch-x86_64 \
-                    --enable-appbundle \
-                    --with-libgcrypt=${prefix} \
-                    --with-openssl=no
+                    --with-openssl=no \
```

Comment:
if the dependency is there anyway, why not set this to "yes"?

Reply:
I have no objections, I just do not know what that gonna entail, so kept non-breaking choices as they were.